azerothcore/azerothcore-wotlk
# fix(Core/SpellEffects): Make Shadowmeld drop combat

## CHANGES PROPOSED:
-  Make Shadowmeld drop combat in open world
-  

## ISSUES ADDRESSED:
- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/3266



## TESTS PERFORMED:
- Builds
- Tested in-game



## HOW TO TEST THE CHANGES:

1. Aggro a mob
2. Use Shadowmeld (`.cast 58984`)
3. See that you lost combat and aggro
4. Use 2 characters and go into an instance and start a boss fight
5. Use Shadowmeld (`.cast 58984`)
6. See that you do not lose combat

## Target branch(es):








 
## How to test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR
